src/Serilog.Sinks.Loki/LokiBatchFormatter.cs

```diff
@@ -43,9 +48,13 @@ public void Format(IEnumerable<LogEvent> logEvents, ITextFormatter formatter, Te
                 content.Streams.Add(stream);
 
                 stream.Labels.Add(new LokiLabel("level", GetLevel(logEvent.Level)));
-                foreach (LokiLabel globalLabel in _globalLabels)
+                foreach (LokiLabel globalLabel in GlobalLabels)
                     stream.Labels.Add(new LokiLabel(globalLabel.Key, globalLabel.Value));
 
+                var localTime = DateTime.Now;
+                var localTimeAndOffset = new DateTimeOffset(localTime, TimeZoneInfo.Local.GetUtcOffset(localTime));
+                var time = localTimeAndOffset.ToString("o");
```

Comment:
Use `var time = logEvent.Timestamp.ToString("o");` instead to use the timestamp from the log event.